fraudserv/service.go

```diff
@@ -138,6 +143,16 @@ func (f *ProofService) Broadcast(ctx context.Context, p fraud.Proof) error {
 	return t.Publish(ctx, bin)
 }
 
+func (f *ProofService) AddVerifier(proofType fraud.ProofType, verifier fraud.Verifier) error {
```

Comment:
Let's add it to `Subscriber` interface with some description. Worth mentioning that it's optional, and the main validity is determined via Validate on the proof itself